composer.json

```diff
@@ -136,7 +136,7 @@
         "rector/rector": "0.18.1",
         "spiral/code-style": "^1.1",
         "spiral/nyholm-bridge": "^1.2",
-        "spiral/testing": "^2.7",
+        "spiral/testing": "dev-feature/scopes as 2.8.0",
```

Comment:
@msmakouz what do we need to release spiral/testing 2.8.0?

Reply:
In the **feature/scopes** branch, the `TestScope` attribute has been added.

Before the release, it is necessary to check the `TestScope` attribute together with the `Config` attribute. There were issues with this, the configuration was being changed twice, causing an exception.

Reply:
Released.